src/interfaces/ManagesImages.java

```diff
@@ -22,24 +22,25 @@
 	/**
 	 * Returns the position of the positive image.
 	 * 
-	 * @param imageList
+	 *
 	 * @return
 	 */
-	public int getPositiveImagePosition(int[] imageList);
+	public int getPositiveImagePosition();
 
 	/**
 	 * Set image indexes.
-	 * 
-	 * @param imageList
+	 *
 	 */
-	public void setIndexImages(int[] imageList);
+	public void setIndexImages();
 
 	/**
 	 * Shuffle image indexes.
 	 *
-	 * @param imageList
 	 * @param totalImages
 	 */
-	public void shuffleIndexImages(int[] imageList, int totalImages);
+	public void shuffleIndexImages(int totalImages);
+
+
 
+	int[] returnArray();
```

Comment:
Pentru claritate, poti pune modificatorul de acces _public_ la aceasta metoda.
Totusi, poti citi pen internet despre conventia in Java. Se prea poate ca in Java conventia sa fie fara modificatori de acces, din cauza ca se considera toate metodele publice din interiorul unei interfete.